### DISCUSSION TITLE
官方frp客户端无法使用

### DISCUSSION THREAD

**Discussion by senssic** [natfrp/wiki#74] [使用问题]
1.下载官方frp 版本0.43.0 https://github.com/fatedier/frp/releases/download/v0.43.0/frp_0.43.0_windows_amd64.zip
2.复制隧道配置文件到 frpc.ini
![image](https://user-images.githubusercontent.com/3918554/172088224-da03726b-5c7b-499e-97ec-e3a5e94d6930.png)
3.使用frpc -c frpc.ini启动客户端，目前启动正常，且隧道状态为在线
![image](https://user-images.githubusercontent.com/3918554/172088315-0b128e72-9752-414d-b7b0-ffc6633d9c26.png)

但浏览器访问就是无法映射成功。
![image](https://user-images.githubusercontent.com/3918554/172088371-d2bf37c4-d0e0-4ee6-b51a-136e7f2d9c24.png)

**Comment by fakeboboliu** [comment]
经测试无兼容性问题，请检查自己应用配置。

另：打码鬼才了属于是，建议重置访问密钥
